tests/kernel/install_klp_product.pm

```diff
@@ -0,0 +1,59 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This module installs the kernel livepatching product and
+#          verifies the installation.
+# Maintainer: Nicolai Stange <nstange@suse.de>
+
+use 5.018;
+use warnings;
+use strict;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+use klp;
+use power_action_utils 'power_action';
+
+sub run {
+    my $self = shift;
+    my $kver;
+    my $kflavor;
+
+    select_console('root-console');
```

Comment:
You don't need to switch terminals here, `boot_ltp` already selected a root terminal for you. You're switching from serial terminal back to VNC console here which means a significant performance penalty.

Reply:
Will do, thanks for the hint

Reply:
Done.